### PR TITLE
Fishing map/latest data available from datasets. MAP-1160

### PR DESCRIPTION
**Pull request by j8seangel**
Fixes https://github.com/GlobalFishingWatch/frontend/pull/2012 using endDate of datasets active in the workspace

### CODE REVIEW

File: apps/fishing-map/features/datasets/datasets.utils.ts [hunk 1]
```diff
@@ -247,6 +247,20 @@ export const getActiveDatasetsInActivityDataviews = (
   })
 }
 
+export const getLatestEndDateFromDatasets = (
+  datasets: Dataset[],
+  datasetCategory?: DatasetCategory
+): string => {
+  if (!datasets.length) return ''
```
Comment (rdgfuentes) [inline apps/fishing-map/features/datasets/datasets.utils.ts:254]: What if there's no visible datasets by default? It'll return `''`, is that expected? Shouldn't it return `DEFAULT_TIME_RANGE.end` in this case to ensure it returns a proper date?

File: apps/fishing-map/data/config.ts [hunk 2]
```diff
@@ -51,12 +51,10 @@ export const FULL_SUFIX = 'full'
 export const USER_SUFIX = 'user'
 export const PRIVATE_SUFIX = 'private'
 
+export const DEFAULT_DATA_DELAY_DAYS = 3
```
Comment (rdgfuentes) [inline apps/fishing-map/data/config.ts:54]: 👍

File: apps/fishing-map/features/app/app.selectors.ts [hunk 3]
```diff
@@ -67,6 +72,24 @@ export const selectTimeRange = createSelector(
   }
 )
 
+export const selectLatestAvailableDataDate = createSelector(
```
Comment (rdgfuentes) [inline apps/fishing-map/features/app/app.selectors.ts:75]: 💯

File: apps/fishing-map/features/datasets/datasets.utils.ts (hunk 1 above)
Comment (j8seangel) [reply apps/fishing-map/features/datasets/datasets.utils.ts]: Good catch! thanks

**Pull request merged**